### PR TITLE
fix(Avatar): Allow Avatar.Group to set skeleton prop

### PR DESCRIPTION
**Pull request by langz**
This PR just adds the property and type for skeleton to the Avatar.Group, so that it will be easier/possible to use the component with the skeleton prop. The functionality is/was already in place, take a look at https://eufemia.dnb.no/uilib/components/avatar and add the skeleton property to one of the demos/examples.

```
<Avatar.Group skeleton label="label">
   <Avatar>A</Avatar>
</Avatar.Group>
 ```

### CODE REVIEW

File: packages/dnb-eufemia/src/components/avatar/AvatarGroup.tsx [hunk 1]
```diff
@@ -46,6 +47,12 @@ export interface AvatarGroupProps {
    * Default: primary.
    */
   variant?: AvatarVariants
+
+  /**
+   * Skeleton should be applied when loading content
+   * Default: null
+   */
+  skeleton?: SkeletonShow
```
Comment (langz) [inline packages/dnb-eufemia/src/components/avatar/AvatarGroup.tsx:55]: Not sure if/how I should use this prop in the component.
As it doesn't seem like I actually need to make more code changes to be able to do:

```
<Avatar.Group skeleton label="label">
   <Avatar>A</Avatar>
</Avatar.Group>
 ```

File: packages/dnb-eufemia/src/components/avatar/AvatarGroup.tsx [hunk 2]
```diff
@@ -55,6 +62,7 @@ export const defaultProps = {
   size: 'medium',
   children: null,
   variant: 'primary',
+  skeleton: null,
```
Comment (dinarosv) [inline packages/dnb-eufemia/src/components/avatar/AvatarGroup.tsx:65]:
```suggestion
  skeleton: false,
```

**Review by dinarosv — CHANGES_REQUESTED** [review]

File: packages/dnb-eufemia/src/components/avatar/AvatarGroup.tsx (hunk 2 above)
Comment (langz) [reply packages/dnb-eufemia/src/components/avatar/AvatarGroup.tsx]: Done ✅

**Review by dinarosv — APPROVED** [review]

**Comment by langz** [comment]
Not really sure what we've done now, when running eufemia locally, I get the following error when visiting http://localhost:8000/uilib/components/avatar/demos

```
Warning: Received `false` for a non-boolean attribute `skeleton`.
If you want to write it to the DOM, pass a string instead: skeleton="false" or skeleton={value.toString()}.
If you used to conditionally omit it with skeleton={condition && value}, pass skeleton={condition ? value : undefined} instead.
```

I'll try looking into it tomorrow.

**Comment by langz** [comment]
What I wanted to achieve with this PR was the following:
image

At the moment, it's not working ,and I am not exactly sure why, but it seems to "work again", if I remove the following line https://github.com/dnbexperience/eufemia/pull/1347/commits/d2622c5022eaccc1b7e7c188e588507df4b31c7c#r826913486

**Comment by dinarosv** [comment]
> At the moment, it's not working when passing skeleton to Avatar.Group, and I am not exactly sure why, but it seems to "magically work again" if I remove the following line [#1347 (comment)](https://github.com/dnbexperience/eufemia/pull/1347#discussion_r826913486)

I think you might need to pass down `skeleton` from AvatarGroup to the clone function for the Avatar? Or another option is to include it in the AvatarGroup context?

**Comment by langz** [comment]
> > At the moment, it's not working when passing skeleton to Avatar.Group, and I am not exactly sure why, but it seems to "magically work again" if I remove the following line [#1347 (comment)](https://github.com/dnbexperience/eufemia/pull/1347#discussion_r826913486)
> 
> I think you might need to pass down `skeleton` from AvatarGroup to the clone function for the Avatar? Or another option is to include it in the AvatarGroup context .... edit: what if you change the order of `{skeleton: context?.skeleton}` and `avatargroupcontext` in Avatar?

Thanks, that did it 🙇 🙏

**Pull request merged**